src/egui/mod.rs

```diff
@@ -1,7 +1,7 @@
 use std::path::PathBuf;
 use std::time::Duration;
 
-use eframe::egui::{self, Button};
+use eframe::egui::{self};
```

Comment:
I wasn't going to nitpick this, but since I got another comment...
```suggestion
use eframe::egui;
```

Reply:
Done